docs/source/packages/fetch.md

```diff
@@ -57,7 +57,7 @@ async function postData (url, data) {
       const data = await response.json();
       return response(null, data);
     } catch (err) {
-      return response(err, null);
+      return res.status(500).send({status: false, msg: err.message});
```

Comment:
This doesn't look correct. Have you tried to run this sample? Where is `res` coming from?

Reply:
res is same like response